cumulus/polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -72,24 +74,7 @@ impl BridgeHubRuntimeType {
 	pub const ID_PREFIX: &'static str = "bridge-hub";
 
 	pub fn chain_spec_from_json_file(&self, path: PathBuf) -> Result<Box<dyn ChainSpec>, String> {
```

Comment:
We don't need this function anymore?

Reply:
[a457c64](https://github.com/paritytech/polkadot-sdk/pull/2457/commits/a457c64a84962fe0e9419947053d9ea3c01b1241)